src/ui/Facet/FacetSearchElement.ts

```diff
@@ -194,6 +194,20 @@ export class FacetSearchElement {
       ).el
     );
     this.input.setAttribute('aria-activedescendant', this.facetValueNotFoundId);
+    this.facetSearch.updateAriaLive(l('NoValuesFound'));
+  }
+
+  public updateAriaLiveWithResults(inputValue: string, numberOfResults: number, moreValuesToFetch: boolean) {
+    let ariaLiveText =
+      inputValue === ''
+        ? l('ShowingResults', numberOfResults, inputValue, numberOfResults)
+        : l('ShowingResultsWithQuery', numberOfResults, inputValue, numberOfResults);
```

Comment:
Is it possible to reuse placeholder numbers to avoid needing to pass `numberOfResults` twice? e.g.
`Showing {0} results for query {1} which has {0} results`.

Reply:
It's for the plural, without the last parameters it shows something like `result<pl>s</pl>`